bandit/src/query-lambda/build-query.ts

```diff
@@ -11,6 +11,7 @@ export const buildQuery = (
 	const endTimestamp = end.toISOString().replace("T", " ");
 	const startTimestamp = start.toISOString().replace("T", " ");
 	const dateForCurrencyConversionTable = subDays(start, 1); //This table is updated daily  but has a lag of 1 day
+	const channel = (test.channel.toLowerCase() === 'epic') ? 'ACQUISITIONS_EPIC' : 'ACQUISITIONS_SUBSCRIPTIONS_BANNER';
```

Comment:
if this is intended to be banner1 only then it's `ACQUISITIONS_ENGAGEMENT_BANNER`, though perhaps we should support both

Reply:
Have updated for both Banner 1 and Banner 2 tests